src/Util/validateInput.js

```diff
@@ -0,0 +1,60 @@
+import { ERROR_MESSAGE, MENU, MENU_KIND } from './constants';
+
+const validateDate = date => {
+  if (!(date >= 1 && date <= 31)) {
+    throw new Error(ERROR_MESSAGE.INVALID_DATE);
+  }
+  if (typeof date !== 'number' || !Number.isInteger(date)) {
+    throw new Error(ERROR_MESSAGE.INVALID_DATE);
+  }
+};
+
+const validateName = menu => {
+  const checkedName = new Set();
+  menu.forEach(([name]) => {
+    if (!Object.keys(MENU).includes(name)) {
+      throw new Error(ERROR_MESSAGE.INVALID_ORDER);
+    }
+    checkedName.add(name);
+  });
+};
+
+const validateDuplicate = menu => {
+  const checkedName = new Set();
+  menu.forEach(([name]) => {
+    if (checkedName.has(name)) {
+      throw new Error(ERROR_MESSAGE.INVALID_ORDER);
+    }
+    checkedName.add(name);
+  });
+};
+
+const validateOnlyDrink = menu => {
+  const types = new Set(menu.map(([name]) => MENU[name].type));
+  if (types.size === 1 && types.has(MENU_KIND.DRINK)) {
+    throw new Error(ERROR_MESSAGE.INVALID_ORDER);
+  }
+};
+
+const validateCount = menu => {
+  if (!menu.every(([, count]) => count >= 1)) {
+    throw new Error(ERROR_MESSAGE.INVALID_ORDER);
+  }
+};
+
+const validateTotalCount = menu => {
+  const total = menu.reduce((acc, [, count]) => acc + count, 0);
```

Comment:
사용하지 않는 파라미터는 언더바로 표현하는 관습이 있다고합니다!

```suggestion
  const total = menu.reduce((acc, [_, count]) => acc + count, 0);
```

Reply:
에어비앤비 컨벤션에서 지양하는 거 같더라구요.. 그래서 혹시나 해서 뻈는데 사용하는게 좋았던 거 같네요 🤔